src/mailadm/conn.py

```diff
@@ -66,7 +61,15 @@ def config(self):
         items = self.get_config_items()
         if items:
             d = dict(items)
-            d["path_virtual_mailboxes"] = self.path_mailadm_db.parent.joinpath("virtual_mailboxes")
+            # remove deprecated config keys
+            try:
+                del d["vmail_user"]
+            except KeyError:
+                pass
+            try:
+                del d["path_virtual_mailboxes"]
+            except KeyError:
+                pass
```

Comment:
is this for migrating existing installs?  I probably wouldn't go for it.  if anything rather adbise in the readme on how to migrate (basically listing the things we do, but not in great detail, just roughly -- let's not get caught in caring too much for migration at this stage of the game)

Reply:
It threw an error when I upgraded, so we need some form of database migration call. I opted for just ignoring deprecated config entries, because it's easier to downgrade then. I would like to put as little as possible into the README, there is already a lot to consider for admins when they want to use mailadm (fortunately moving from dovecot to mailcow reduces that a lot).

Let's open an issue to remind ourselves of deleting it when there are proper db migrations?

Reply:
i simply wouldn't offer code for migration but have a small section in the readme with some hints and advise people to open issues if they have questions/issues.  Migrating from the intertwined dovecot/postfix/mailadm to a mailcow-docker based setup is anyway about more things than just the config vars.

Reply:
after your recent changes, is this still needed?